<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>602</width>
    <height>481</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>记事本</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QPlainTextEdit" name="plainTextEdit">
      <property name="styleSheet">
       <string notr="true">QPlainTextEdit
{
	border: none;
}</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>602</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_F">
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionNew_Window"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="separator"/>
    <addaction name="actionPrefrence"/>
    <addaction name="actionPrint"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu_E">
    <property name="title">
     <string>编辑(&amp;E)</string>
    </property>
    <addaction name="actionUndo_U"/>
    <addaction name="separator"/>
    <addaction name="actionCut_T"/>
    <addaction name="actionCopy_C"/>
    <addaction name="actionPaste_P"/>
    <addaction name="actionDelete_L"/>
    <addaction name="separator"/>
    <addaction name="actionSearch_By_Bing"/>
    <addaction name="actionFind_F"/>
    <addaction name="actionFind_Next_N"/>
    <addaction name="actionFind_Prev_V"/>
    <addaction name="actionReplace_R"/>
    <addaction name="actionGoto_G"/>
    <addaction name="separator"/>
    <addaction name="actionSelect_All_A"/>
    <addaction name="actionTime_Date_D"/>
   </widget>
   <widget class="QMenu" name="menu_O">
    <property name="title">
     <string>格式(&amp;O)</string>
    </property>
    <addaction name="actionWord_Wrap_W"/>
    <addaction name="actionFont_F"/>
   </widget>
   <widget class="QMenu" name="menu_V">
    <property name="title">
     <string>查看(&amp;V)</string>
    </property>
    <widget class="QMenu" name="menu_Z">
     <property name="title">
      <string>缩放(&amp;Z)</string>
     </property>
     <addaction name="actionZoom_In_I"/>
     <addaction name="actionZoom_Out_O"/>
     <addaction name="actionZoom_Default"/>
    </widget>
    <addaction name="menu_Z"/>
    <addaction name="actionStatus_Bar_S"/>
   </widget>
   <widget class="QMenu" name="menu_H">
    <property name="title">
     <string>帮助(&amp;H)</string>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="actionFeedback_F"/>
    <addaction name="separator"/>
    <addaction name="actionAbout_A"/>
   </widget>
   <addaction name="menu_F"/>
   <addaction name="menu_E"/>
   <addaction name="menu_O"/>
   <addaction name="menu_V"/>
   <addaction name="menu_H"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew">
   <property name="text">
    <string>新建(&amp;N)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionNew_Window">
   <property name="text">
    <string>新窗口(&amp;W)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>打开(&amp;O)...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>保存(&amp;S)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>另存为(&amp;A)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionPrefrence">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>页面设置(&amp;U)</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>打印(&amp;P)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>退出(&amp;X)</string>
   </property>
  </action>
  <action name="actionUndo_U">
   <property name="text">
    <string>撤销(&amp;U)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionCut_T">
   <property name="text">
    <string>剪切(&amp;T)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionCopy_C">
   <property name="text">
    <string>复制(&amp;C)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionPaste_P">
   <property name="text">
    <string>粘贴(&amp;P)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionDelete_L">
   <property name="text">
    <string>删除(&amp;L)</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionSearch_By_Bing">
   <property name="text">
    <string>使用 Bing 搜索...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionFind_F">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>查找(&amp;F)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionFind_Next_N">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>查找下一个(&amp;N)</string>
   </property>
   <property name="shortcut">
    <string>F3</string>
   </property>
  </action>
  <action name="actionFind_Prev_V">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>查找上一个(&amp;V)</string>
   </property>
   <property name="shortcut">
    <string>Shift+F3</string>
   </property>
  </action>
  <action name="actionReplace_R">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>替换(&amp;R)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="actionGoto_G">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>转到(&amp;G)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="actionSelect_All_A">
   <property name="text">
    <string>全选(&amp;A)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionTime_Date_D">
   <property name="text">
    <string>时间/日期(&amp;D)</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionFont_F">
   <property name="text">
    <string>字体(&amp;F)</string>
   </property>
  </action>
  <action name="actionStatus_Bar_S">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>状态栏(&amp;S)</string>
   </property>
  </action>
  <action name="actionZoom_In_I">
   <property name="text">
    <string>放大(&amp;I)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+=</string>
   </property>
  </action>
  <action name="actionZoom_Out_O">
   <property name="text">
    <string>缩小(&amp;O)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
  <action name="actionZoom_Default">
   <property name="text">
    <string>恢复默认缩放</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+0</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>查看帮助(&amp;H)</string>
   </property>
  </action>
  <action name="actionFeedback_F">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>发送反馈(&amp;F)</string>
   </property>
  </action>
  <action name="actionAbout_A">
   <property name="text">
    <string>关于记事本(&amp;A)</string>
   </property>
  </action>
  <action name="actionWord_Wrap_W">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>自动换行(&amp;W)</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
